x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/numeric_type.tsx

```diff
@@ -44,6 +44,45 @@ interface Props {
 export const NumericType = ({ field }: Props) => {
   return (
     <>
+      {/* scaling_factor only applies to scaled_float numeric type*/}
+      <FormDataProvider pathsToWatch="subType">
+        {formData =>
+          formData.subType === 'scaled_float' ? (
+            <EditFieldSection>
+              <EditFieldFormRow
+                title={
+                  <h3>
+                    {i18n.translate('xpack.idxMgmt.mappingsEditor.scalingFactorFieldTitle', {
+                      defaultMessage: 'Scaling factor',
+                    })}
+                  </h3>
+                }
+                description={i18n.translate(
+                  'xpack.idxMgmt.mappingsEditor.scalingFactorFieldDescription',
+                  {
+                    defaultMessage: 'The scaling factor to use when encoding values.',
```

Comment:
Seems a little strange that this text is different than the text you see when you first create the field. Can we make them the same?